assemble/src/main/scripts/accumulo-client-build.jsh

```diff
@@ -0,0 +1,53 @@
+/*
+# Licensed to the Apache Software Foundation (ASF) under one or more
+# contributor license agreements.  See the NOTICE file distributed with
+# this work for additional information regarding copyright ownership.
+# The ASF licenses this file to You under the Apache License, Version 2.0
+# (the "License"); you may not use this file except in compliance with
+# the License.  You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+*/
+  String accumuloProp = 
+      System.getProperty("user.dir").replace("/bin", "/conf/accumulo-client.properties");
```

Comment:
If possible it would be nice to find the client props file on the classpath.  For the following suggestion I copied code from [here](https://github.com/apache/accumulo/blob/aab34a980d7a9ccad63797913b99b331c5cbbaae/core/src/main/java/org/apache/accumulo/core/cli/ClientOpts.java#L207-L216) and modified it a bit.

```suggestion
  URL clientPropsUrl =
          AccumuloClient.class.getClassLoader().getResource("accumulo-client.properties");
      if (clientPropsUrl != null) {
        accumuloProp = clientPropsUrl.getFile();
      }
```

Not sure if this workable though.  It partially depends on how the classpath is setup for accumulo jshell.  If that follows how other java processes are launched, then the conf dir would be on the classpath.  Also not sure if classloader stuff works in jshell.  If it does work it would give the user more control over where the client props file is found because they can easily change the classpath in accumulo-env.sh.

Reply:
Thank you for the helpful suggestions @keith-turner. I'll spend time trying to implement your suggestions and will update you on how it goes soon. As for the classpath, I believe your idea could work since the jshell-init.jsh contains the default public APIs needed to execute Accumulo tasks.

Reply:
I agree that discovering it from the class path would be better, especially since that will work without making any assumptions about the directory layout from which you are executing the script.

Reply:
@keith-turner Your classpath idea to find the client props appears to be working in JShell. I was able to auto-generate a client and use that client to interact with Accumulo using the classpath method.